studio/components/interfaces/Organization/BillingSettingsV2/Subscription/PlanUpdateSidePanel.tsx

```diff
@@ -289,23 +305,93 @@ const PlanUpdateSidePanel = () => {
       <Modal
         loading={isUpdating}
         alignFooter="right"
-        className="!w-[450px]"
+        className="!w-[550px]"
         visible={selectedTier !== undefined && selectedTier !== 'tier_free'}
         onCancel={() => setSelectedTier(undefined)}
         onConfirm={onUpdateSubscription}
         overlayClassName="pointer-events-none"
-        header={`Confirm ${planMeta?.change_type === 'downgrade' ? 'downgrade' : 'upgrade'} to ${subscriptionPlanMeta?.name}`}
+        header={`Confirm ${planMeta?.change_type === 'downgrade' ? 'downgrade' : 'upgrade'} to ${
+          subscriptionPlanMeta?.name
+        }`}
       >
+        <Modal.Content className="mt-5">
+          {subscriptionPreviewIsLoading && (
+            <span className="text-sm">Estimating monthly costs...</span>
+          )}
+          {!subscriptionPreviewIsLoading && subscriptionPreview && (
```

Comment:
use isSuccess parameter from the RQ here, so we won't need to check loading nor if subscription Preview is undefined or not